src/inventory/BaseInventory.php

```diff
@@ -23,6 +23,7 @@
 
 namespace pocketmine\inventory;
 
+use Closure;
```

Comment:
Global namespace classes are usually referred to with a `\` instead of importing